app/src/main/java/com/example/android/moviedbtrainingapp/presenter/adapter/MovieAdapter.java

```diff
@@ -1,88 +0,0 @@
-package com.example.android.moviedbtrainingapp.presenter.adapter;
-
-
-import android.content.Context;
-import android.support.v7.widget.RecyclerView;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.bumptech.glide.Glide;
-import com.example.android.moviedbtrainingapp.R;
-import com.example.android.moviedbtrainingapp.model.general.Movie;
-
-import java.util.Collections;
-import java.util.List;
-
-import static com.example.android.moviedbtrainingapp.model.utils.Constants.BACK_DROP;
-import static com.example.android.moviedbtrainingapp.model.utils.NumberRounder.twoDecimalNumbers;
-
-public class MovieAdapter extends RecyclerView.Adapter<MovieAdapter.MovieHolder> {
-
-    LayoutInflater inflater;
-    Context context;
-    List<Movie> movies = Collections.EMPTY_LIST;
-    MovieClickListener movieListener;
-
-    public MovieAdapter(Context context, MovieClickListener listener) {
```

Comment:
Instead of Context and MovieClickListener being passed (both of them are actually the same object right now!) you can just pass MainActivity and keep reference to it.